assets/css/site/logo.css

```diff
@@ -4,6 +4,8 @@
 }
 .logo svg {
   display: flex;
+	width: 36px;
```

Comment:
@bastianallgeier alternatively to setting all svg to that size, we could wrap all those inserted via `icon()` helper with a span that sets this size.